getting-started/troubleshooting/error-handling.rst

```diff
@@ -283,20 +313,21 @@ in the dashboard:
    Click on the red Remove icon to remove a transfer or SIP from the dashboard,
    then click Confirm.
 
-It is recommended that you clear your dashboard of transfers and SIPs periodically
-to improve browser performance.
+It is recommended that you clear your dashboard of transfers and SIPs
+periodically to improve browser performance.
 
 .. _error-browser:
 
 Browser compatability
 ---------------------
 
-Archivematica has been tested most extensively with Firefox and Chrome. There are
-known issues with Internet Explorer 11 which result in an inability to start
-transfers in the dashboard (`Issue 7246`_).  Minimal, but successful,
-testing has been done with Microsoft Edge.
+Archivematica has been tested most extensively with Firefox and Chrome. There
+are known issues with Internet Explorer 11 which result in an inability to start
+transfers in the dashboard (`Issue 7246`_).  Minimal, but successful, testing
```

Comment:
```suggestion
transfers in the dashboard (`Issue 7246`_). Minimal, but successful, testing
```
Ooooh I thought I'd caught all the period+doublespace occurrences in the docs but every once in a while I find one!